packages/code-du-travail-data/indexing/document.mapping.js

```diff
@@ -2,6 +2,10 @@ export const documentMapping = {
   properties: {
     description: { type: "text" },
     metaDescription: { type: "text" },
+    // only in thematic files
+    categories: {
+      enabled: false,
```

Comment:
what the purpose of `enabled: false` ?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/enabled.html
It disables the parsing of items so you can fetch the field of course, but elastic does strictly nothing to/with it.